packages/stk/stk_ngp/stk_ngp/NgpField.hpp

```diff
@@ -379,6 +379,14 @@ class StaticField : public FieldBase
 
     const stk::mesh::BulkData& get_bulk() const { return *hostBulk; }
 
+    STK_FUNCTION
+    void swap_data(StaticField<T> &sf)
```

Comment:
Why not just call this "swap"?

Reply:
Yes, swap would be just as good here. Maybe there's even a better name. The intent is to do a "shallow" swap. I.e., in traditional sierra-framework fields it was just a pointer swap, not a copying of the data. It is for managing multi-state fields, e.g., an 'n' and 'n+1' state.

Reply:
@alanw0 It looks like this means the same thing as, say, `std::shared_ptr` swap:

https://en.cppreference.com/w/cpp/memory/shared_ptr/swap

If so, then I think `swap` is the right name for this function.

Reply:
done.